### DISCUSSION TITLE
Alist挂载的云盘失效，报错53: connect: invalid argument

### DISCUSSION THREAD

**Discussion by Ho-YuChen** [AlistGo/alist#6885] [Q&A]
之前一直可以正常挂载115、百度、阿里、夸克网盘并在nas上通过wedav读取，最近两天nas重启后重新挂载外部硬盘发现均挂载不上，AList存储页面均显示无法通过qrcode进行认证，并且尝试使用cookie认证也出现同样53报错，求大佬们指点一下🙏
<img width="750" alt="image" src="https://github.com/user-attachments/assets/b288d1e9-7182-4762-80da-606c5975da01">

**Comment by yzqpc** [comment]
我第一次使用，用二维码扫码获取token挂载115，也是出现问题：

failed to login by qrcode: {"state":0,"error":"登录失败，系统已下架！如果你有电脑端的使用需求，我们诚挚邀请你下载体验115产品专属客户端“115浏览器”或在线使用“115网页端(115.com)”，畅享智能高效云生活。","errno":0,"message":"登录失败，系统已下架！如果你有电脑端的使用需求，我们诚挚邀请你下载体验115产品专属客户端“115浏览器”或在线使用“115网页端(115.com)”，畅享智能高效云生活。","code":0}: unexpected error

在115 app上也没有显示到有网页版登陆的情形，难道大家新登录的都也有这样的问题吗？怎么解决呢？

**Comment by Simonzhang8** [comment]
网络问题？
